modulegen/dependabot_test.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"os"
+	"path/filepath"
+	"runtime"
+	"slices"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/testcontainers/testcontainers-go/modulegen/internal/context"
+	"github.com/testcontainers/testcontainers-go/modulegen/internal/dependabot"
+)
+
+// isWindows returns if the current OS is Windows. For that it checks the GOOS environment variable or the runtime.GOOS constant.
+func isWindows() bool {
+	return os.Getenv("GOOS") == "windows" || runtime.GOOS == "windows"
+}
+
+func TestGetDependabotConfigFile(t *testing.T) {
+	if isWindows() {
+		t.Skip("skipping on windows because the dependabot config file uses the '/' separator")
```

Comment:
question: not sure I understand, what's the issue here?

I'm concerned that if the tests don't work the the generation wouldn't work either.

Reply:
I'm reverting it, so that the tests will check the unix-separator. My intention was to not run the tests for our files, which are sanity checks for the format of the current state of the dependabot file.

Reply:
Test passing even on Windows 😉